Sources/KIF/Classes/KIFUITestActor.m

```diff
@@ -399,6 +399,19 @@ - (void)swipeFromEdge:(UIRectEdge)edge
     }];
 }
 
+-(void)drawSignature;
+{
+    UIView *signaturePad = nil;
+    UIAccessibilityElement *element = nil;
+    [self waitForAccessibilityElement:&element view:&signaturePad withLabel:@"Signature Pad" value:nil traits:UIAccessibilityTraitAllowsDirectInteraction tappable:YES];
```

Comment:
This feels like an app specific concern. This seems like something that should be added as a class extension within your tests bundle rather than something that needs to be added directly to KIF itself.

FYI - The PR linked is to an internal repo, not one that is publicly available.

Reply:
Do you have any examples of that? I was having trouble with SSH so used codespaces but see that didn't quite work out how I was hoping